www/docs/customization/homebrew_casks.md

```diff
@@ -311,27 +323,41 @@ homebrew_casks:
   - name: foo
     custom_block: |
       module GitHubHelper
-        def self.get_asset_api_url(tag, name)
+        def self.github_token
```

Comment:
i would name this `token`, as its inside a GitHub helper module already

Reply:
Sorry, I fixed it